lib/stdlib/src/math.erl

```diff
@@ -134,8 +134,31 @@ tan(_) ->
 tanh(_) ->
     erlang:nif_error(undef).
 
+-spec ceilf(X) -> float() when
+      X :: number().
+ceilf(_) ->
+    erlang:nif_error(undef).
+
+-spec floorf(X) -> float() when
+      X :: number().
+floorf(_) ->
+    erlang:nif_error(undef).
+
 %%% End of BIFs
 
+-spec ceil(X) -> integer() when
```

Comment:
Hmm have you checked what you measure?

The code becomes:
```

{function, test_ceil_1, 0, 2}.
  {label,1}.
    {line,[{location,"math_test.erl",10}]}.
    {func_info,{atom,math_test},{atom,test_ceil_1},0}.
  {label,2}.
    {move,{integer,2},{x,0}}.
    return.


{function, test_ceil_2, 0, 4}.
  {label,3}.
    {line,[{location,"math_test.erl",20}]}.
    {func_info,{atom,math_test},{atom,test_ceil_2},0}.
  {label,4}.
    {move,{integer,1},{x,0}}.
    return.
```


Though I can not give a reason why it takes longer, probably gc since you don't spawn the functions.

This discussion is moot this PR as suggested will probably not be used anyway.